docs/Developers/Git_Basics.rst

```diff
@@ -283,6 +283,57 @@ another editor will be fired up with all the messages of commits which you want
 to squash. Clean up all the messages and add a new message to be
 displayed for the single commit.
 
+Common Git Issues
+-----------------
+
+Sometimes, you use git add-A and add files you didn't want to your push (often
+after rebasing) and push it to the remote. Here ,is a short outline of, how can
+you remove (or revert changes in) particular files from your commit even after
+pushing to remote.
+
+In your local repo, to revert the file to the state before the previous commit
+run the following:
+::
+
+    $ git checkout HEAD^ /path/to/file
+
+Now , after reverting the file(s) update your last commit , by running :
+::
+
+    $ git commit --amend
+
+To apply these changes to the remote you need to force update the branch :
+::
+
+    $ git push -f myfork
+
+.. note::
+
+    The upstated procedure helps roll back changes by one commit only.
```

Comment:
upstated procedure -> procedure outlined above
(gah, sorry I missed, it, it was hidden by a massive comment chain.